docker-compose.review.yml

```diff
@@ -0,0 +1,53 @@
+version: '3.4'
+services:
+  redis:
+    image: redis:3.2-alpine
+    restart: unless-stopped
+  postgres:
+    image: mdillon/postgis:9.6-alpine
+    environment:
+      POSTGRES_PASSWORD: "${POSTGRES_PASSWORD:-password}"
+      POSTGRES_USER: "${POSTGRES_USER:-postgres}"
+    volumes:
+      - db-data:/var/lib/postgresql/data:cached
+    ports:
+      - "54320:5432"
+    restart: unless-stopped
+  vets-api:
+    build:
+      context: .
+      target: development
```

Comment:
👍 👍 vets-api has a number of `Rails.env` checks in it, but I dont think any that we cant work through. Happy to see this.